snap/snapcraft.yaml

```diff
@@ -52,16 +52,16 @@ parts:
   cleanup:
     after: [mattermost-desktop]
     plugin: nil
-    build-snaps: [ gnome-3-28-1804 ]
+    build-snaps: [ gnome-3-38-2004 ]
     override-prime: |
         set -eux
-        cd /snap/gnome-3-28-1804/current
+        cd /snap/gnome-3-38-2004/current
         find . -type f,l -exec rm -f $SNAPCRAFT_PRIME/{} \;
 
 apps:
   mattermost-desktop:
-    extensions: [gnome-3-28]
-    command: opt/Mattermost/mattermost-desktop --no-sandbox
```

Comment:
The `--no-sandbox` flag was causing this later version to segfault when it started.